apps/k8s/ksonnet/myapp/environments/default/.metadata/k.libsonnet

```diff
@@ -0,0 +1,80 @@
+local k8s = import "k8s.libsonnet";
```

Comment:
`.metadata` seems to indicate this directory may not have to be included?

Reply:
The files inside this folder are pretty big. Do they need to be included?

Reply:
The versions of ksonnet < 0.9.0, there is a .metadata folder which holds ksonnet-lib. This has been relocated in the current version. I can work with @christopherhein to get this project into shape.